README.adoc

```diff
@@ -1,34 +1,125 @@
-////
-This is the base template for Hazelcast integration module guides.
-
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
-
-:github-address: [github-repo-address-of-your-guide]
+:github-address: https://github.com/hazelcast-guides/getting-started-hazelcast-springboot
 :templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+:hazelcast: Hazelcast IMDG
+:framework: Spring Boot
 
-= Title
+= Getting Started with Hazelcast using Spring Boot
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+This guide will help you to start using Hazelcast by building a Spring Boot application.
+
+include::{templates-url}/what-is-hz.adoc[]
 
 include::{templates-url}/link-to-repo.adoc[]
 
 == What You’ll Learn
 
-// Define the problem and tell reader what he/she will learn from this guide.
+include::{templates-url}/microservices/what-you-will-learn.adoc[]
 
 == Prerequisites
 
-// Define the prerequisites for the guide.
+- ~15 minutes
+include::{templates-url}/microservices/prerequisites.adoc[]
+
+== Using Hazelcast in the Spring Boot Application
+
+include::{templates-url}/microservices/application-structure.adoc[]
+
+If Hazelcast is on the classpath and a suitable configuration is found, Spring Boot auto-configures a HazelcastInstance that you can inject in your application. In the `pom.xml` file, you can see Hazelcast is added as a dependency:
+
+[source,xml]
```

Comment:
What about gradle users?

Reply:
Yes, we should add support for gradle users.